File: 251001/Brutskaya/disCo/publisher/src/main/java/com/example/publisher/api/dto/request/IssueRequestTo.java
package com.example.publisher.api.dto.request;

import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class IssueRequestTo {

    private Long id;

    private Long userId;

    @Size(min = 2, max = 64)
    private String title;

    @Size(min = 4, max = 2048)
    private String content;
}

File: 251003/Pankratiev/rest/publisher/src/main/kotlin/bsuir/dc/publisher/dto/from/IssueRequestTo.kt
package bsuir.dc.publisher.dto.from

import jakarta.validation.constraints.Size

data class IssueRequestTo(
    val id: Long = 0,
    val writerId: Long = 0,
    val labels: List<String> = listOf(),

    @field:Size(min = 2, max = 64)
    val title: String = "",

    @field:Size(min = 4, max = 2048)
    val content: String = ""
)
